src/config/config.json

```diff
@@ -0,0 +1,12 @@
+{
+  "email": "test@test.com",
+  "pipeline_user": "sw23",
+  "pipeline_passwd": "fake",
+  "pipeline_hostname": "localhost",
+  "pipeline_db_name": "aperature",
+  "columns": {
+    "vehicle_number": { "max": "NA", "min": 0 },
+    "maximum_speed": { "max": 150, "min": 0 },
+    "service_date": { "max": "NA", "min": "1990-01-01" }
+  }
+}
```

Comment:
I'm curious what your thoughts would be on having this file in assets/ instead.

Reply:
That makes sense to me, I'll move it.